Header header
int32 n
Leaf[] leafs